common/webapp/src/commonMain/kotlin/models/BaseRequest.kt

```diff
@@ -23,6 +24,11 @@ interface BaseRequest<R> {
     val resultSerializer: KSerializer<R>
 }
 
+interface SimpleRequest : BaseRequest<Unit> {
```

Comment:
It is better to name it `UnitRequest` due to its returning type

Reply:
This request will only be used for requests without a response body, they will only be expected to report the success or failure of the request to the client